src/libsyntax/ext/expand.rs

```diff
@@ -70,10 +70,13 @@ pub fn expand_expr(e: P<ast::Expr>, fld: &mut MacroExpander) -> P<ast::Expr> {
 
         ast::ExprKind::InPlace(placer, value_expr) => {
             // Ensure feature-gate is enabled
-            feature_gate::check_for_placement_in(
-                fld.cx.ecfg.features,
-                &fld.cx.parse_sess.span_diagnostic,
-                expr_span);
+            if let Some(&Features { pushpop_unsafe: false, .. }) =
+                fld.cx.ecfg.features {
+                feature_gate::emit_feature_err(
+                    &fld.cx.parse_sess.span_diagnostic, "pushpop_unsafe", expr_span,
```

Comment:
Yeah I saw that but I didn't have the time to fix it earlier... I copy-pasted too fast. Sorry about that.